spec/status.yml

```diff
@@ -96,7 +96,7 @@ components:
       get:
```

Comment:
To be consistent with other endpoints, can you also update the URL for this endpoint to be `/{authenticator}/{service_id}/{account}/status`?

Maybe we should have a central definition for `service_id`?

Reply:
I added a new reference-able object for the service_id parameter, should make things more readable